app/services/create_repository_service.rb

```diff
@@ -0,0 +1,32 @@
+class CreateRepositoryService
```

Comment:
It's a factory. You could call it `RepositoryFactory` instead, which seems nicer IMO.

Reply:
I agree with you about factories, but this is following the service pattern in rails. Because this pattern was also used in happy-customer, the same pattern was repeated.

Reply:
It's just a naming thing. The fact that some other project uses services shouldn't prevent you from calling this a factory.